std_msgs/Header header
Polygon3D[] polygons
